pkg/logs/client/http/destination.go

```diff
@@ -84,7 +98,26 @@ func errorToTag(err error) string {
 
 // Send sends a payload over HTTP,
 // the error returned can be retryable and it is the responsibility of the callee to retry.
-func (d *Destination) Send(payload []byte) (err error) {
+func (d *Destination) Send(payload []byte) error {
+	if d.blockedUntil.After(time.Now()) {
+		log.Debugf("%s: sleeping until %v before retrying", d.url, d.blockedUntil)
+		d.waitForBackoff()
+	}
+
+	err := d.send(payload)
+
+	if _, ok := err.(*client.RetryableError); ok {
+		d.nbErrors = d.backoff.IncError(d.nbErrors)
+	} else {
+		d.nbErrors = d.backoff.DecError(d.nbErrors)
+	}
+
+	d.blockedUntil = time.Now().Add(d.backoff.GetBackoffDuration(d.nbErrors))
+
+	return err
+}
+
+func (d *Destination) send(payload []byte) (err error) {
```

Comment:
NIT: maybe we could rename this so it's clear how its different than the public `Send`? perhapse `unconditionalSend`? 
(It took me a moment to figure out that `SendAsync` does not respect the backoff policy and I think renaming this function would make it much more clear)